h2o-core/src/main/java/water/jdbc/SQLManager.java

```diff
@@ -73,6 +79,7 @@ SELECT ROW_NUMBER() OVER () AS RowNum_, <table>.* FROM <table>
         rs = stmt.executeQuery("SELECT COUNT(1) FROM " + table);
         rs.next();
         numRow = rs.getLong(1);
+        rs.close();
```

Comment:
good catch!

Reply:
In fact, this one is required for Netezza driver to work, as some drivers do not support more than one connection at a time. We could do a conn.getMetadata()... check, but in this case such check is completely in vain and the resultSet can be closed right away.